src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -880,27 +875,10 @@ export class ActivityFeedItem extends React.Component<Props> {
           };
         } else {
           const additionalInfo = {};
-          const isTrxBetweenAccounts = (isKWAddress(event.to, accounts) && isSWAddress(event.from, accounts)) ||
-            (isKWAddress(event.from, accounts) && isSWAddress(event.to, accounts));
-
-          const sendingAccountType = isReceived
-            ? (findAccountByAddress(event.to, accounts) || {}).type
-            : (findAccountByAddress(event.from, accounts) || {}).type;
-
-          if (isTrxBetweenAccounts) {
-            if (isForAllAccounts) {
-              additionalInfo.itemImageSource = sendingAccountType === ACCOUNT_TYPES.KEY_BASED
-                ? keyWalletIcon
-                : smartWalletIcon;
-            } else {
-              additionalInfo.itemImageSource = event.accountType === ACCOUNT_TYPES.KEY_BASED
-                ? keyWalletIcon
-                : smartWalletIcon;
-            }
-            additionalInfo.isBetweenAccounts = true;
-          }
 
-          if (!isTrxBetweenAccounts) {
+          if (isBetweenSmartWalletAccounts) {
```

Comment:
Similar to one of the previous comments, this seems to replace KW <-> SW check for SW <-> SW check. Pls make sure this is intended.

Reply:
KW is not supported, all KW code are leftovers (except migration).